tgui/packages/tgui/interfaces/BlueSpaceArtilleryControl.js

```diff
@@ -0,0 +1,55 @@
+import { useBackend } from "../backend";
+import { Button, LabeledList, Section, Box, ProgressBar } from "../components";
+import { Window } from "../layouts";
+
+export const BlueSpaceArtilleryControl = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window>
+      <Window.Content>
+        <Section>
+          <LabeledList>
+            {data.notice && (
+              <LabeledList.Item label="Alert" color="red">
+                {data.notice}
+              </LabeledList.Item>
+            )}
+            <LabeledList.Item label="Target">
+              <Button
+                icon="crosshairs"
+                content={data.target ? data.target : "None"}
+                onClick={() => act("recalibrate")} />
+            </LabeledList.Item>
+            {data.ready === 1 && !!data.target && (
+              <LabeledList.Item label="Firing">
+                <Button
+                  icon="burn"
+                  content={"FIRE!"}
+                  color={"red"}
```

Comment:
No need to evaluate anything here, just do `content="FIRE!"`

Even better would be to put it as the button's contents:
```jsx
<Button
	icon="burn"
	color="red"
	onClick={() => act("fire")}>
	FIRE!
</Button>
```

Reply:
`content="FIRE!` is the standard TGUI convention, as used in the examples.
```jsx
        <Button content="Simple" />
        <Button selected content="Selected" />
        <Button altSelected content="Alt Selected" />
        <Button disabled content="Disabled" />
        <Button color="transparent" content="Transparent" />
        <Button icon="cog" content="Icon" />
        <Button icon="power-off" />
        <Button fluid content="Fluid" />
```